server/intelligence-service/app/mentor/mentor.py

```
@@ -0,0 +1,87 @@
+"""
+LangGraph-based mentor implementation.
+"""
+import json
+from typing import Dict, Any, List
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)